### DISCUSSION TITLE
What is the AWS charge "Hubs Cloud Personal" and why did it go up?

### DISCUSSION THREAD

**Discussion by wmurphyrd** [mozilla/hubs#3709] [Hubs Cloud Questions] [answered]
![image](https://user-images.githubusercontent.com/10034859/104541720-d0927e80-55e7-11eb-9b61-1f7a3ccbf7f3.png)
What's the charge for "Hubs Cloud Personal" - it seems all the AWS resources are accounted for separately (compute - EC2, database - RDS, storage - S3), but there's an additional charge that's listed as type "software usage" and service "Hubs Cloud Personal."

Is this a fee charged by Mozilla? If so, why did it increase sharply to ~$150 / month starting January 1st?

**Comment by robinkwilson** [comment, marked as answer]
That's correct! Yes, that is the fee for using the software for Hubs Cloud and scales with the EC2 size. 

On this page, if you scroll down, you can see the pricing differences - _see the screenshot below_: https://aws.amazon.com/marketplace/pp/B084RZH56R

So for running an EC2 instance for running the software:
**$0.014 / hr** for a **t3.medium _and below_**
**$0.10 / hr** for a **t3.large _and above_**
If you're running a multi-server setup, it's per EC2 machine. Personal, you have only 1 server.

When you turn your HC stack to [Offline mode](https://hubs.mozilla.com/docs/hubs-cloud-aws-costs.html#offline-mode---manual) with a [stack update](https://hubs.mozilla.com/docs/hubs-cloud-aws-updating-the-stack.html) (aka not running any EC2 instances), you will not be paying for running the software then - or the db costs.

Hope this helps!

![image](https://user-images.githubusercontent.com/29560735/104596935-89c57880-5629-11eb-862b-1751e43bd1d3.png)

**Comment by robinkwilson** [comment]
@wmurphyrd Just to double check:

Does your cost chart coincide with the EC2 instance size scaled up? We want to make sure there wasn't anything out of the ordinary here.

**Reply by wmurphyrd** [reply]
@robinkwilson yes it did coincide with a EC2 change form medium to large on my 2 hubs clouds (I think I forgot what the original size was when scaling back after an event :sweat_smile: ). Thanks!
